### ISSUE TITLE
Has this project been abandoned?

### ISSUE THREAD

**Issue by dav01d** [EpicGamesExt/BlenderTools#737]

**Comment by thevioletdev** [comment]
Seems like it :/

**Comment by vikista** [comment]
:(

**Comment by hhzx** [comment]
Why？:(

**Comment by Xiarno** [comment]
Since the last version supported is Blender 3.6.0, which is an LTS version, it's possible that they are waiting for Blender 4.2 which is also an LTS version before releasing a new update to this addon.

The target for 4.2 LTS release is currently July 16th.
Source: https://projects.blender.org/blender/blender/milestone/19

**Comment by DanMcLaughlin** [comment]
Yes it's been discussed elsewhere, James left the company and there's nobody supporting this anymore. But the license was changed and we're now allowed to fork and develop it ourselves. What is needed is somebody to take the lead on that.

**Comment by Xiarno** [comment]
> Yes it's been discussed elsewhere, James left the company and there's nobody supporting this anymore. But the license was changed and we're now allowed to fork and develop it ourselves. What is needed is somebody to take the lead on that.

Is that so? Well, if a fork get made that is usable and updated, would you mind linking it here please?

**Comment by DanMcLaughlin** [comment]
If you read my comment I said somebody _needs_ to take the lead.

**Comment by Xiarno** [comment]
> If you read my comment I said somebody _needs_ to take the lead.

And if you read my comment, I said "if a fork get made", which is in the future, whenever that would be, if you could please link it here.

**Comment by DanMcLaughlin** [comment]
I'm trying to tell you I'm not taking control of this project and am unlikely to have any part of it, perhaps you could take the lead and find somebody who is willing to take responsibility on this.

**Comment by JoshQuake** [comment]
I'd love to do so to keep it alive, however I'm not skilled enough to do major fixes. I'd have to rely on others for most updates. However that being said I can do so and began implementing the pending pulls into my own fork.

If other contributors are okay with that, the fork is here [https://github.com/JoshQuake/BlenderTools](url).

**Comment by DanMcLaughlin** [comment]
That's great @JoshQuake! I think all we need is a maintainer who can 

- Integrate all the pending and existing fixes, there are two are three already, the LOD regex error, the Blender 4.0 FBX update (older) and there might be one more
- Merge new fixes from the community
- Publish releases

**Comment by jellb** [comment]
Please keep it alive 🙏

**Comment by JoshQuake** [comment]
> That's great @JoshQuake! I think all we need is a maintainer who can
> 
> * Integrate all the pending and existing fixes, there are two are three already, the LOD regex error, the Blender 4.0 FBX update (older) and there might be one more
> * Merge new fixes from the community
> * Publish releases
> 
> However that link doesn't go anywhere, is the repo public? Are you able do these things and continue maintenance?
> 
> Edit: some URL parsing problems, if I go to your home page and view it from there looks like you already cut a release

oops fixed link. Yeah I'm definitely down to do that. This tool is too useful to be abandoned.

**Comment by TactileVisions** [comment]
Thank you, @JoshQuake

**Comment by dav01d** [comment]
Awesome work @JoshQuake 

In case people missed it, Josh is now running this project in a fork: https://github.com/JoshQuake/BlenderTools

**Comment by dav01d** [comment]
Closing thread for now, see above 👆

**Comment by dav01d** [comment]
Actually, I realize I should keep this issue open, incase google, or other visitors come by. It will be the very first thing people see. Apologies for the reopen, but its purely for guidance of future users :)

**Comment by jellb** [comment]
What about us 2 rigify? Or it's included in the fork?

**Comment by JoshQuake** [comment]
> What about ue 2 rigify? Or it's included in the fork?

It's included. As far as I'm aware it doesnt need an update yet as the checks pass.

**Comment by jellb** [comment]
> > What about ue 2 rigify? Or it's included in the fork?
> 
> It's included. As far as I'm aware it doesnt need an update yet as the checks pass.
Great. For the ue2rigify,
there are no unreal 5 mannequins presets.  So metarig should be updated

**Comment by JoshQuake** [comment]
> > > What about ue 2 rigify? Or it's included in the fork?
> > 
> > 
> > It's included. As far as I'm aware it doesnt need an update yet as the checks pass.
> 
> Great. For the ue2rigify, there are no unreal 5 mannequins presets. So metarig should be updated

okok. I shall take a look this weekend and see what I can do.

**Comment by jack-yao91** [comment]
Hi I am a little late to the conversation, but @JoshQuake thanks for stepping up and helping out with this project. I am a tool developer, and we have become very reliant on this addon being a part of our pipeline. Me and a co-worker had created our own fork here and integrated some of the PRs and stood up the CI/CD workflows for testing, docs, and creating releases. 
https://github.com/poly-hammer/BlenderTools

Send to Unreal is passing all the tests in Blender 4.1, however UE to Rifigy is still [failing some tests here](https://github.com/poly-hammer/BlenderTools/actions/runs/9489301811). (which look like some changes to the rigify API)

Our primary interest in keeping this addon working and up to date, is so that users of our other tools can rely on the integrations we have made with send2ue.

Our org has a self-hosted runner pool we use for other repos that we were able to utilize for our BlenderTools fork, but maybe we could discuss a way to combine efforts? You have shown a great deal of ownership with project, so I definantly don't want to slow you down, but maybe we can figure out how to share some of our runners with you. I can follow up with some more details here.
https://github.com/JoshQuake/BlenderTools/issues/16

**Comment by Derjyn** [comment]
Okay. So which one do we utilize? It's confusing... The 2 repositories that are active, point fingers at each other:

https://github.com/JoshQuake/BlenderTools
https://github.com/poly-hammer/BlenderTools

Can the maintainers of those update their docs to reflect which is which, and we can get a final confirmation on which is the legitimate, active, end-all-be-all repo?

---

**_JoshQuake repo:_**

![image](https://github.com/EpicGamesExt/BlenderTools/assets/648059/9a1bfc6e-5082-49a8-a3ac-6fc8f5fad023)

**_Poly-Hammer repo:_**

![image](https://github.com/EpicGamesExt/BlenderTools/assets/648059/50e69c86-3c8d-4307-bae4-081dc502df74)

**Comment by JoshQuake** [comment]
Poly-Hammer is the new home. I will be archiving my fork when everything is migrated.

**Comment by Derjyn** [comment]
> Poly-Hammer is the new home. I will be archiving my fork when everything is migrated.

Awesome! Thanks for the quick reply. I hit a bit of speedbump when working on internal documentation concerning production pipelines, and wasn't expecting the Epic side to go dead, and then got more confused when finding that you guys picked up the banner.

Thanks for all your hard work. I can get back to the joys of development docs :smirk:
